package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "code-oss-dev",
   "version": "1.52.0",
-  "distro": "e289d0d793bde8f8b5632625e01c50c218e7e127",
+  "distro": "bedb1d87b28d90d8cf83deb8f334816a05980fb4",
```

Comment:
This is just to fix conflicts with distro and get exploration builds out from this branch, will remove this before merging and fix conflicts on distro branch.